src/image.tsx

```diff
@@ -1,47 +1,84 @@
+import classnames from 'classnames';
 import * as React from 'react';
 import {createUseStyles} from './jss';
 
+/**
+ * This context is used internally to disable the border radius. This is useful for example
+ * when using the Image component inside a Card
+ */
+const DisableBorderRadiusContext = React.createContext(false);
+
+export const useDisableBorderRadius = (): boolean => React.useContext(DisableBorderRadiusContext);
+
+export const DisableBorderRadiusProvider: React.FC = ({children}) => (
+    <DisableBorderRadiusContext.Provider value>{children}</DisableBorderRadiusContext.Provider>
+);
+
 const useStyles = createUseStyles(() => ({
     image: {
-        borderRadius: 4,
+        borderRadius: ({noBorderRadius}) => (noBorderRadius ? 0 : 4),
         display: 'block',
         objectFit: 'cover',
         maxWidth: '100%',
         maxHeight: '100%',
+        aspectRatio: ({aspectRatio}) => aspectRatio ?? 'unset',
```

Comment:
Note that this CSS property is only supported by modern browsers.

For browsers where this property is not supported (and no height or width are given), the media will be displayed in the original aspect ratio, which is good enough to me